Build.cmake

```diff
@@ -85,6 +85,10 @@ ENDIF()
    ELSE()
          list(REMOVE_ITEM SRC_FILES  ${LOG_SRC}/crashhandler_windows.cpp ${LOG_SRC}/g3log/stacktrace_windows.hpp ${LOG_SRC}/stacktrace_windows.cpp)
    ENDIF (MSVC OR MINGW)
+   
+   IF (MSVC) 
+         SET(CMAKE_DEBUG_POSTFIX "_d")
```

Comment:
for fun I tried to set this for Linux (commenting out the MSVC check). It failed. 

This explains how it should be set but I have not yet tried it. I will try to see if this (see link) actually does anything or if the link below has to be adhered to: 

http://stackoverflow.com/questions/28893450/how-to-get-debug-postfix-in-executable-name